man/journald.conf.xml

```diff
@@ -106,11 +106,14 @@
 
       <varlistentry>
         <term><varname>Compress=</varname></term>
+        <term><varname>CompressThresholdBytes=</varname></term>
```

Comment:
hmm, thinking about this, I think it would make sense to just merge these two settings into one, i.e. if a boolean value is specified then we'll turn compression off or on with the default threshold. If however it is specified as a numeric value then we'll turn it on and set the threshold accordingly. I think I'd prefer it that way since the setting is quite specialist, and I think it would fit quite naturally into the concept. I am generally afraid of exposing too many options and if we can fold two into one this way this would be preferable.

of course, internally it should be treated the way you already do, all I am suggesting is that in the configuration file we merge this into one setting. Or to say this differently: the existing Compress option would just gain a new set of valid values.